go.mod

```diff
@@ -49,14 +49,15 @@ require (
 	k8s.io/cri-api v0.27.0-alpha.3
 )
 
-require github.com/freddierice/go-losetup v0.0.0-20220711213114-2a14873012db // indirect
+require github.com/freddierice/go-losetup v0.0.0-20220711213114-2a14873012db
```

Comment:
Can we merge this into the above `require`?

Reply:
Fixed. And also moved some direct mods to the area above.